src/components/layout/_Sidebar.svelte

```diff
@@ -8,20 +8,26 @@
 	}
 </script>
 
-<AppRail gap="gap-2" width="w-20" height="h-full" border="">
+<AppRail
```

Comment:
Hier kann man die Sidebar eigentlich auch dynamisch machen und sie mit der aus der Landingpage verheiraten. Eventuell macht man halt dann zwei Komponenten (logged in bzw nicht logged in) und zeigt sie dann entsprechend an. Alternativ kann man auch ein Array aus den Komponenten mit jeweiligen Flag, wann sie angezeigt werden soll, deklarieren...

Reply:
Darüber hinaus ist die Sidebar in der mobile Ansicht nicht linksbündig (da sieht man noch ein Abstand) und sie scheint zu hoch zu sein, wegen der Scrollbar

Reply:
Die Sidebar für das Layout wird noch mit bisschen mehr Logik versehen als die angemeldete Version, daher würde ich die beiden auch getrennt lassen. Allgemeine Thematik zu Sidebar: Wird morgen oder übermorgen angepasst